docs/guide/concept-configurations.md

```diff
@@ -97,7 +97,7 @@ configuration file for the [Basic Project Template](start-installation.md).
 $config = [
     'id' => 'basic',
     'basePath' => dirname(__DIR__),
-    'extensions' => require(__DIR__ . '/../vendor/yiisoft/extensions.php'),
+    'extensions' => require __DIR__ . '/../vendor/yiisoft/extensions.php',
```

Comment:
Imo the function syntax is easier to read in cases where `require` is used to return something...